eos-autoupdater/main.c

```diff
@@ -193,6 +224,24 @@ info (const gchar *msgid,
   va_end (args);
 }
 
+static gint
+mkdir_print_errors (const gchar *path,
+                    gint         mode)
+{
+  gint ret = g_mkdir_with_parents (path, mode);
+
+  if (ret != 0) {
```

Comment:
Nitpick: Coding style is to have braces 2-spaces indented on the following line. I think there are a few places where the autoupdater doesn’t comply with this style (which is confusing), but we might as well tidy them up when touching them.

Same in a few places in this commit.

Reply:
Good catch. I agree with making the style consistent when in the area.